packages/fuels/tests/types/generics/src/main.sw

```diff
@@ -25,7 +25,7 @@ enum EnumWGeneric<N> {
 
 struct MegaExample<T, U> {
     a: ([U; 2], T),
-    b: ([EnumWGeneric<StructWTupleGeneric<StructWArrayGeneric<PassTheGenericOn<T>>>>; 1], u32),
+    b: Vec<([EnumWGeneric<StructWTupleGeneric<StructWArrayGeneric<PassTheGenericOn<T>>>>; 1], u32)>,
```

Comment:
😆

Reply:
A PITA to maintain, but I do feel safe seeing it pass.